### DISCUSSION TITLE
Pump chip reference

### DISCUSSION THREAD

**Discussion by SigmaPic** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#327] [General]
Hello,

I'm looking to identify the reference of the chip that manage the signals on the pump.

Someone know a bit more about that logic ?

![136789766-c43aaba9-92ff-4629-9687-ae5ffebc5394~2.jpg](https://user-images.githubusercontent.com/2431226/178135143-56628c98-eb5e-4d56-af31-4095afa98822.jpg)

**Comment by menime8** [comment]
I have a Nuvoton MS51FB9AE there.

**Reply by visualapproach** [reply]
God job! I looked at the datasheet but got no wiser. They describe the SPI timings and stuff, but hard to know what they done in software. So we want to know if we need any filters. Can you check the display side also? Is there 1k resistors with capacitors there too?
What happens if you overclock the ESP to 160 MHz? (Edit platformio.ini)

**Reply by menime8** [reply]
I think we should stay in one thread with my problem.  :-)

I see pins that go through the sealant. Its for testing or programming. Any chance to get the code out of it? Upload to PC? Or is this programming everytime one way?

**Reply by visualapproach** [reply]
Possibly hackable but don't know

**Reply by SigmaPic** [reply]
> I think we should stay in one thread with my problem.  :-)


Yep. The question was not related to your issue but more globally how are managed the com signals.

Yes maybe hackable but this project allow to do a lot of things.
